src/its_preselector/controlbyweb_web_relay.py

```diff
@@ -77,7 +81,15 @@ def get_digital_input_value(self, input_num: int) -> bool:
             digital_input = root.find(input_tag)
         if digital_input is None:
             raise ConfigurationException(f"Digital Input {input_num} does not exist.")
-        return bool(digital_input.text)
```

Comment:
I think this needs to be digital_input.text == "1". Also. I think you could just do bool(int(digital_input.text))